### PR TITLE
Make tests use `parseFileSync` instead of raw loader objects

### PR DESCRIPTION
**Pull request by ibgreen**

### CODE REVIEW

File: modules/core/src/file-utils/load-file.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export function loadFile(url, loaders, options) {
 
 export function loadFileSync(url, loaders, options) {
   const data = readFileSync(url, options);
-  const result = parseFileSync(data, url, loaders, options);
+  const result = parseFileSync(data, loaders, options, url);
```
Comment (gnavvy) [inline modules/core/src/file-utils/load-file.js]: Add a comment on when we should use `url`?

File: modules/ply/src/ply-loader.js [hunk 2]
```diff
@@ -14,8 +14,7 @@ const DEFAULT_OPTIONS = {
 export default {
   name: 'PLY',
   extension: 'ply',
-  format: 'text',
-  parseBinary: parsePLY,
-  parseText: parsePLY,
+  parseTextSync: parsePLY,
+  parseSync: parsePLY,
```
Comment (gnavvy) [inline modules/ply/src/ply-loader.js]: We have separate parsers for e.g. `parseBinaryGLTF` and `parseTextGLTF`. 
Shall we split `parsePLY` to `parseBinaryPLY` and `parseTextPLY` or at least add a comment that both formats are handled inside `parsePLY`?

File: modules/core/src/file-utils/load-file.js (hunk 1 above)
Comment (ibgreen) [reply modules/core/src/file-utils/load-file.js]: Better yet, I updated the docs

File: modules/ply/src/ply-loader.js (hunk 2 above)
Comment (ibgreen) [reply modules/ply/src/ply-loader.js]: Done

**Pull request merged**